roles/setup-slack/tasks/invite_users.yml

```diff
@@ -7,8 +7,24 @@
   set_fact:
     channels_ids: "{{ user.channels | map('extract', channel_mapping) | list }}"
 
+- name: Create user_create payload
+  set_fact:
+    user_payload:
+      channels: "{{ channels_ids|join(',') }}"
```

Comment:
Could the `join` be done at line 8 above to avoid processing this content more than once?